src/hotspot/share/gc/shenandoah/shenandoahEvacTracker.cpp

```diff
@@ -89,24 +106,20 @@ void ShenandoahEvacuationTracker::print_evacuations_on(outputStream* st,
   mutators->print_on(st);
   st->cr();
 
-  AgeTable young_region_ages(false), old_region_ages(false);
   ShenandoahHeap* heap = ShenandoahHeap::heap();
-  for (uint i = 0; i < heap->num_regions(); ++i) {
-    ShenandoahHeapRegion* r = heap->get_region(i);
-    if (r->age() > 0 && r->age() < AgeTable::table_size) {
```

Comment:
It turns out that this was previously needed because one could not bound the region age. For now, I am dropping this test and instead bounding region ages, clamping them at `markWord::max_age`; see `ShenandoahHeapRegion::increment_age()`.

Should we reach a situation where we need to use larger ages than this, we'll need to revisit this code.

At this time, region ages pose a number of issues including injection of noise into object demographics and mortality rate calculations which drive our adaptive tenuring algorithm.